src/Application.php

```diff
@@ -35,10 +38,34 @@ class Application extends BaseApplication
      */
     public function bootstrap(): void
     {
+        // Call parent to load bootstrap from files.
         parent::bootstrap();
 
+        if (PHP_SAPI === 'cli') {
+            $this->bootstrapCli();
+        }
+
+        /*
+         * Only try to load DebugKit in development mode
+         * Debug Kit should not be installed on a production system
+         */
+        if (Configure::read('debug')) {
+            $this->addPlugin('DebugKit');
```

Comment:
since `DebugKit` plugin is in `required-dev` (like `Bake` plugin) we should allow `addPlugin` failure (catching `MissingPluginException` probably)